pages_ru/rails/200_real_apps/020_logging.md

```diff
@@ -95,9 +95,11 @@ annotations:
 
 Теперь попробуем развернуть приложение:
 ```shell
-werf converge --repo <имя пользователя Docker Hub>/werf-guide-app
+werf converge --repo <ИМЯ ПОЛЬЗОВАТЕЛЯ Docker Hub>/werf-guide-app
 ```
 
+> Убедитесь, что правильно указали имя пользователя от Docker Hub.
```

Comment:
Don't think this is necessary. Or we should make a note like this every time we ask the user to provide any input. No one does this.